code/__HELPERS/shell.dm

```diff
@@ -0,0 +1,56 @@
+//Runs the command in the system's shell, returns a list of (error code, stdout, stderr)
```

Comment:
This file deals with the ```shell()``` proc, thus it has the potential to carry a shell injection exploit. I've done my best, but please be extra thorough when reviewing the code here.